.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+<!--
+Please follow the instructions below before filing a new issue.
+
+Please ensure that you are at least on the newest minor version of the SDK.
+The minor version is delineated in the following example with an `x`: `0.x.11`
+
+There are many changes to the Swift language and Xcode that are not backwards compatible
+and older versions of Swift and Xcode are not guaranteed to be supported by the Contentful Swift team.
```

Comment:
> older versions of Swift and Xcode
Older than what?

> not guaranteed to be supported
Sounds weird, why not just "not supported".

> Contentful Swift team
Just "Contentful"?